packages/core/src/types.ts

```diff
@@ -56,8 +56,14 @@ export type TransactionProps = {
   payerAddress: PublicKey;
 };
 
+// ATA = Associated Token Address
+export type ATAAllowOwnerOffCurve = {
```

Comment:
I believe the name is too complicated. Is it important to duplicate part '''allowOwerOffCurve" both in type and as a field?

Reply:
It's not actually, but I didn't find more proper name for it, because this type relates directly to `allowOwerOffCurve`.

We can name it something like `StakeAdditionalProps` ?

Reply:
Or 
```ts
type StakeCommonProps = {
  amount: number;
  allowOwerOffCurve?: boolean;
}
```
then, we'll use it in `StakeProps` and `getStakeTransaction`.

Reply:
Maybe like `TransactionAdditionalProps`?

Reply:
It is not transaction props